venom-tests/lib/applyManifest.yml

```diff
@@ -0,0 +1,12 @@
+executor: applyManifests
+input:
+  filename: {}
+steps: 
+  - type: exec
+    script: |
+      kubectl apply -f '{{.input.filename}}'
+      sleep 20
```

Comment:
it looks okay. 
I am not sure but can we use `kubectl wait` command if `sleep` is used to wait for the resource to be Ready?

Reply:
Sounds like a good ideea if we don't want to always lose those 20 secs.

Reply:
Great idea!

Reply:
@buraksekili I am not able to find a condition for `kubectl wait` to check if resource was created successfully.
Do you know how to do it?

Reply:
it seems like `kubectl wait` is not feasible to use with CRDs. Let's keep this part as it is